spec/features/result_filters/location_spec.rb

```diff
@@ -69,6 +69,35 @@
     end
   end
 
+  describe "filtering by location" do
```

Comment:
This was spec was moved out of the "Selecting an option" describe block (line 130). If you think it reads better doing it this way then it would be worth doing the same for the other tests that are still within the "Selecting an option" block for consistency ...

Reply:
I've just changed the nesting of those tests. I have a feeling the tests themselves should be changed and not just assert against the query string? Let me know what you think. I'll squash before any merge

Reply:
Good point. I guess now the API is hooked up we should be asserting what results should appear on the results page also.

Reply:
I added a new ticket for the extra work https://trello.com/c/FKnZUeeP/300-refactor-results-filter-tests